Blog post for GSoC 2019 Call for Mentors (#1988)
jenkins-infra/jenkins.io: content/blog/2018/12/2018-12-22-gsoc-2019-call-for-mentors.adoc

  gitter: /jenkinsci/gsoc-sig
  googlegroup: jenkins-gsoc-all-public
  sig: gsoc
---

NOTE: link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
is as program where students are paid a stipend by Google to work on a free open source project like Jenkins,
at full-time for four months (May to August).
Mentors get actively involved with students starting at the end of February when students start to apply
(see the link:https://developers.google.com/open-source/gsoc/timeline[timeline]).

image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]

We are looking for mentors and project ideas to participate in the 15th edition of the Google Summer of Code program!
We have until February 6th, 2019 at 8pm UTC to submit the application on behalf the Jenkins Organization, but obviously,
we want to be ready before that.

The first step in the process is to have mentors and project ideas.
Then we will apply to Google.
We need Google to accept Jenkins' application to the program itself.
And for this to happen, *we need project proposals and mentors*.

We currently have a link:/projects/gsoc/2019/project-ideas[list of project idea proposals],

> and are still accepting new project proposals, mentors, technical advisers, and subject matter experts.

Reviewer: ```suggestion
, and we are looking for new project proposals, mentors, technical advisers, and subject matter experts.
```

> 

Reviewer: ```suggestion
GSoC projects may be about anything around code: new features, plugins, test frameworks, infrastructure, etc., etc.

```

> Making a project idea proposal is easy, link:/projects/gsoc/proposing-project-ideas[read the quick start instructions]

Reviewer: Maybe makes sense to just copy-paste steps and put them here so that it takes less time

or ask us on the link:https://gitter.im/jenkinsci/gsoc-sig[GSoC SIG Gitter chat].

Mentoring takes about 5 to 6 hours of work per week (more at the start, less at the end).
In return, a student works on your project full time for four months.
Think about the projects that you've always wanted to do but could not find the time...
GSoC is a pretty good return on the investment!

> Potential mentors are invited to read the link:/projects/gsoc/mentors[information for mentors] which lists the benefits, expectations

Reviewer: Benefits could be put here, and expectations are briefly summarized above. Then you can reference the Mentor guide for more details. I do not think it makes sense to reference conflicts of interest directly
